sdk/dslink/src/main/java/org/dsa/iot/dslink/node/actions/table/Modify.java

```diff
@@ -0,0 +1,81 @@
+package org.dsa.iot.dslink.node.actions.table;
+
+public class Modify {
+	
+	private int removeFrom = -1;
+	private int insertBefore = -1;
+	
+	private Modify(int insertBefore) {
+		this.insertBefore = insertBefore;
+	}
+	
+	private Modify(int removeFrom, int insertBefore) {
+		this.removeFrom = removeFrom;
+		this.insertBefore = insertBefore;
+	}
+	
+	
+	public boolean isReplace() {
+		return (removeFrom > -1 && insertBefore > -1);
+	}
+	
+	public boolean isInsert() {
+		return (removeFrom == -1 && insertBefore > -1);
+	}
+	
+	public int getInsertIndex() {
+		if (isInsert()) {
+			return insertBefore;
+		}
+		throw new UnsupportedOperationException("not an insert");
```

Comment:
Not sure I like these. Is there a way to use polymorphism instead of that? IMO it breaks the API and it's confusing.